integration-tests/tests/snap_find_test.go

```diff
@@ -34,14 +34,16 @@ type searchSuite struct {
 }
 
 func (s *searchSuite) TestSearchMustPrintMatch(c *check.C) {
-	searchOutput := cli.ExecCommand(c, "snap", "find", "hello-world")
-
 	// XXX: Summary is empty atm, waiting for store support
```

Comment:
ok, IMO that should be done in another PR, the aim of this one is adding a partial name search to the test, what do you think?